pages/config/index.tsx

```diff
@@ -0,0 +1,325 @@
+        await axiosWithApiKey(apiKey).get('/api/test_api_key')
+        dispatch(saveApiKey({ apiKey }))
+      }
+      dispatch(updateUseCdnSdk({ useCdnSdk }))
+      router.push('/')
+    },
+  )
+
+  const errorMessage = getSaveApiKeyErrorMessage(error)
+
+  return (
+    <div className="min-h-screen bg-gradient-to-br from-slate-50 via-blue-50 to-indigo-100 flex items-center justify-center p-4">
+      <div className="relative w-full max-w-2xl">
+        {/* Main Card */}
+        <div className="bg-white/80 backdrop-blur-sm rounded-2xl shadow-2xl border border-white/20 overflow-hidden">
+          {/* Header Section */}
+          <div className="bg-gradient-to-r from-indigo-600 to-purple-600 px-8 py-6 text-white">
+            <div className="flex items-center space-x-3 mb-2">
+              <div className="w-10 h-10 bg-white/20 rounded-lg flex items-center justify-center">
+                <img
+                  src="/icons/settings.svg"
+                  alt="Settings"
+                  width={24}
+                  height={24}
+                />
+              </div>
+              <div>
+                <h1 className="text-2xl font-bold tracking-tight">
+                  Configuration
+                </h1>
+                <p className="text-indigo-100 text-sm">
+                  Set up your Kanmon integration
+                </p>
+              </div>
+            </div>
+          </div>
+
+          {/* Form Section */}
+          <div className="p-8">
+            <Formik
+              onSubmit={async (values) => {
+                await saveApiKeyFn(values)
+              }}
+              initialValues={initialValues}
+              validateOnMount={true}
+            >
+              {({
+                isValid,
+                handleSubmit,
+                isSubmitting,
+                values,
+                setFieldValue,
+              }) => {
+                return (
+                  <Form className="space-y-6">
+                    {/* API Key Section */}
+                    <div className="space-y-3">
+                      <div className="flex items-center space-x-2">
+                        <div className="w-8 h-8 bg-indigo-100 rounded-lg flex items-center justify-center">
+                          <img
+                            src="/icons/key.svg"
+                            alt="API Key"
+                            width={16}
+                            height={16}
+                          />
+                        </div>
+                        <label className="text-sm font-semibold text-gray-700">
+                          API Key
+                        </label>
+                      </div>
+                      <FormikTextInput<FormValues>
+                        fieldName="apiKey"
+                        placeholder="Enter your Kanmon API key"
+                        updateContainerCss={basicCssClassUpdater('w-full')}
+                        customInputSx={{
```

Comment:
where is this design coming from? i don't think it matches what we have on other pages